documentation/src/docs/index.md

```diff
@@ -52,21 +52,20 @@ Pravega can be installed locally or in a distributed environment. The installati
 ## Pravega Security, Role-based access control and TLS
 
  Pravega supports encryption of all communication channels and pluggable role-based access control. For more information please refer to the following:
-1. [TLS](security/pravega-security-encryption.md)
-2. [Authorization, authentication and RBAC](security/pravega-security-authorization-authentication.md)
 
+1. [TLS](security/pravega-security-encryption.md)
+2. [Authorization, Authentication and RBAC](security/pravega-security-authorization-authentication.md)
 
 ## Support
 
-Don’t hesitate to ask! Contact the developers and community on the [mailing lists](https://groups.google.com/forum/#!forum/pravega-users) or on [slack](https://pravega-io.slack.com/) if you need any help.
-Please open an issue in [Github
-Issues](https://github.com/pravega/pravega/issues) if you find a bug.
+Don’t hesitate to ask! Contact the developers and community on the [Mailing List](https://groups.google.com/forum/#!forum/pravega-users) or on [Slack](https://pravega-io.slack.com/) if you need any help.
```

Comment:
@fpj 
The above mentioned mailing list connects to google groups,!?Is that Okay?
Would you like to provide any new mail ids??Jus an ask.